rules/concepts.md

```diff
@@ -22,4 +23,59 @@ formerly implicit variables
 How to find and install.
 How to write will be on a later page.
 
-### Comprehensive Examples
\ No newline at end of file
+### Comprehensive Examples
+-->
+
+# Rules Concepts
+
+Thus far we've connected openHAB to devices through Things, modeled the devices with Items, discussed persistence and how to build the display to control your home.
+But all of that amounts to home control, not automation.
+To create home automation we need to define rules.
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+## What Are Rules
+
+You can think of rules as routines or behaviours for your smart home.
+
+Just like you stand up in the morning and have your breakfast, your smart home can have a routine to open the blinds when the sun rises.
+Another routine would be to turn all lights off when you leave home: openHAB can do that for you, no need to open your app and switch of the lights manually.
+You want to hear your favorite music when you arrive back at home? No problem, openHAB can do that for you.
+But rules can also remind you of things: you opened the window hours ago and forgot that is is open? openHAB can sent a notification to your phone.
+
+To summarize in an universal principle: *When __a__ happens, than do __b__*.
+For many cases, it can be practical to extend that to *When __a__ happens, than do __b__ but only if __c__*.
```

Comment:
We could also have a rule that is just `then do __b__`. So I would just present the full abc form and then mention that both __a__ and __c__ can be optional (in the UI a rule without triggers and conditions is a "Script"). 

I really don't like putting the "but only if" at the end because if you think about how the rule is actually evaluated, it flows from triggers to conditions and then to actions.  But it's presented in this order in the UI. I'm not sure which is better here, especially since I don't think most of the languages support conditions when writing text based rules. I'd say use the order that makes the most sense to support how they are presented in this page.

Oh, here's an idea, what if instead of a, b, c we use t, a, and c for triggers, actions, and conditions? I really want to make those names for the parts of the rule clear  and that might help the readers remember them.

NOTE: in this case we are talking about time, i.e. "do __b__ after __a__ happens" so use "then" instead of "than".